CONTRIBUTING.md

```diff
@@ -0,0 +1,237 @@
+```
+
+Upon making this change, run:
+
+```sql
+SELECT pg_reload_conf();
+```
+
+Running the earlier SHOW command should reveal the appropriate change:
+
+```sql
+SHOW vectorize.embedding_service_url;
+```
+```text
+   vectorize.embedding_service_url
+-------------------------------------
+ http://localhost:3000/v1/embeddings
+(1 row)
+```
+
+#### 4.3. Load example data
+
+The following can be found within the this project's README, under [Hugging Face Example](https://github.com/tembo-io/pg_vectorize/blob/main/README.md#hugging-face-example).
+
+Begin by creating a `producs` table with the dataset that comes included with `pg_vectorize`.
+
+```sql
+CREATE TABLE products AS
+SELECT * FROM vectorize.example_products;
+```
+
+You can then confirm everything is correct by running the following:
+
+```sql
+SELECT * FROM products limit 2;
+```
+```text
+ product_id | product_name |                      description                       |        last_updated_at        
+------------+--------------+--------------------------------------------------------+-------------------------------
+          1 | Pencil       | Utensil used for writing and often works best on paper | 2023-07-26 17:20:43.639351-05
+          2 | Laptop Stand | Elevated platform for laptops, enhancing ergonomics    | 2023-07-26 17:20:43.639351-05
+```
+
+#### Sample queries
+
+```sql
+SELECT vectorize.table(
+job_name => 'product_search_hf',
+"table" => 'products',
+primary_key => 'product_id',
+columns => ARRAY['product_name', 'description'],
+transformer => 'sentence-transformers/multi-qa-MiniLM-L6-dot-v1'
+);
+```
+```text
+                    table
+---------------------------------------------
+ Successfully created job: product_search_hf
+(1 row)
+```
+
+```sql
+SELECT * FROM vectorize.search(
+job_name => 'product_search_hf',
+query => 'accessories for mobile devices',
+return_columns => ARRAY['product_id', 'product_name'],
+num_results => 3
+);
+```
+```text
+                                       search_results
+---------------------------------------------------------------------------------------------
+ {"product_id": 13, "product_name": "Phone Charger", "similarity_score": 0.8147812194590133}
+ {"product_id": 6, "product_name": "Backpack", "similarity_score": 0.774306211384604}
+ {"product_id": 11, "product_name": "Stylus Pen", "similarity_score": 0.7709903789778251}
+(3 rows)
+```
+
+### 5. Local URL
+
+Once all of the following is complete, you should be able to visit the `Tembo-Embedding-Service` at [http://localhost:3000/docs](http://localhost:3000/docs) and explore.
+This is a platform that allows, for example, the input of [different sentence-transformers models](https://huggingface.co/models?sort=trending&search=sentence-transformers) from Hugging Face.
+
+# Packaging
+
+Run this script to package into a `.deb` file, which can be installed on Ubuntu.
+
+```
+/bin/bash build-extension.sh
+```
```

Comment:
Where is this script?

Reply:
Deleted section for now. Found inspiration in pgmq contrib:

https://github.com/tembo-io/pgmq/blob/e00e716b1a79ec330de2060720cee8425c5c4355/CONTRIBUTING.md?plain=1#L88